src/main/java/jalau/at18/kingoftokyo/RandomDice.java

```diff
@@ -0,0 +1,28 @@
+package jalau.at18.kingoftokyo;
+
+import java.util.Random;
+
+public class RandomDice {
+    private int randomDice;
+
+    public Dice generatedRandom() {
+        randomDice = new Random().nextInt(DiceFace.values().length) + 1;
+        for (DiceFace element : DiceFace.values()) {
+            if (element.getRepresentativeValue() == randomDice) {
+                Dice dice = new Dice(element);
+                return dice;
+            }
+        }
+        return null;
```

Comment:
Do we ever reach this line? if not, we should not use this line

Reply:
I don't need it anymore after changing the code